### PR TITLE
feat: updated rsk endpoints to use sovryn endpoints

### PR DESCRIPTION
**Pull request by bradleySuira**
#Linear Ticket

[https://linear.app/liquality/issue/LIQ-123/evaluate-moving-to-sovryns-endpoints-503-node-error](https://linear.app/liquality/issue/LIQ-123/evaluate-moving-to-sovryns-endpoints-503-node-error)

### CODE REVIEW

**Review by beemi — APPROVED** [review]
LGTM

File: src/store/factory/client.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ function createSolanaClient (network, mnemonic, derivationPath) {
 function createRskClient (asset, network, mnemonic, accountType, derivationPath) {
   const isTestnet = network === 'testnet'
   const rskNetwork = ChainNetworks.rsk[network]
-  const rpcApi = isTestnet ? 'https://public-node.testnet.rsk.co' : 'https://public-node.rsk.co'
+  const rpcApi = isTestnet ? 'https://testnet.sovryn.app/rpc' : 'https://mainnet.sovryn.app/rpc'
```
Comment (monokh) [inline src/store/factory/client.js:180]: There are also other occurances of this in the wallet, please update / consolidate

**Review by monokh — CHANGES_REQUESTED** [review]

File: src/store/factory/client.js (hunk 1 above)
Comment (bradleySuira) [reply src/store/factory/client.js:180]: got it, let me check right now
Thanks!

Comment (bradleySuira) [reply src/store/factory/client.js]: @monokh done, it's fixed now, added an env var to set mainnet and testnet endpoints then if we need to update in the future only the env var should be updated

**Review by beemi — APPROVED** [review]

**Pull request merged**